### ISSUE TITLE
Make paradox-menu-mode work better with proportional fonts

### ISSUE THREAD

**Issue by hexmode** [Malabarba/paradox#143]
"Archive" and "Description" columns line up.  "Version" and "Status" should, too.
![image](https://user-images.githubusercontent.com/43581/27554220-5aa6b268-5a73-11e7-900e-754e27c583f4.png)

**Comment by Malabarba** [comment]
Maybe paradox does something wrong here, but I think it might be a bug with package.el or tabulated-list-mode. Can you reproduce this if you restart Emacs and launch the regular package manager (with M-x list-packages)?

**Comment by hexmode** [comment]
Side by side, two emacs started with "emacs -q --no-site-file".
![image](https://user-images.githubusercontent.com/43581/27733810-bda15124-5d5d-11e7-870d-88d9beba11ed.png)

I set the default face in both to google's [Go font](https://blog.golang.org/go-fonts)

I did "M-x package-init RET" to load paladox and skipped the github account setup on the left.

**Comment by Malabarba** [comment]
Thanks, indeed it's some issue with paradox. It may take me a bit of time to look into it, so any help is welcome.

**Comment by hexmode** [comment]
Just noticed something that might be helpful. 
![image](https://user-images.githubusercontent.com/43581/28242470-87d9effa-697a-11e7-933d-ee4fc3742c2e.png)

Package lines without the "h" (indicating a homepage) *are* lined up.

Package lines with the "h" *are not* lined up.

**Comment by Malabarba** [comment]
I'm sorry it took me so long to think of this. But you should be able to workaround the problem by disabling `paradox-use-homepage-buttons`.

**Comment by hexmode** [comment]
Thanks!  That workaround does the trick.

I think it also shows that paradox needs to add a column for the home page link just as one has been added for github stars.

**Comment by Malabarba** [comment]
True. I initially implemented it this way to save space. But I think it deserves its own column now.

**Comment by Malabarba** [comment]
PRs welcome. :-)
